### DISCUSSION TITLE
Flet

### DISCUSSION THREAD

**Discussion by Siva-Rajan-R** [flet-dev/flet#3024] [General]
How to pack and build a native python  packages in flet ...

**Comment by ndonkoHenri** [comment]
For which platform?

**Reply by Siva-Rajan-R** [reply]
In,Windows

**Comment by ndonkoHenri** [comment]
- new and recommended: https://flet.dev/docs/publish/windows
- old: https://flet.dev/docs/cookbook/packaging-desktop-app

**Reply by Siva-Rajan-R** [reply]
Bro how to do this for android apk ,i already lot of times seen flet documents and i tried that methods but can't understand clearly please if you don't mind can you make a video for how to build apk with native python pacakges on your YouTube channel...its my request🙂 for all platforms...its may usefull for many flet developers
